components/search/TierToggler.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div>
+    <p class="font-italic">
+      {{ toggleText }}
+      <nuxt-link
```

Comment:
We have used `<b-link>` on several other components, which will use a `<nuxt-link>` if you specific the to.  
https://bootstrap-vue.js.org/docs/components/link/

I am inclined to say: keep it consistent and use the `<b-link>` here as well.